modules/nw-route-specific-annotations.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * networking/routes/route-configuration.adoc
+
+[id="nw-route-specific-annotations_{context}"]
+= Route-specific annotations
+
+The Ingress Controller can set the default options for all the routes it exposes. An individual route can override some of these defaults by providing specific configurations in its annotations.
```

Comment:
@ironcladlou @danehans we had a request to bring this content forward for 4.x. I made some minor updates to it, and included @lihongan's suggestions. Would you mind also verifying?

Thanks!

Also - since we've switched from router to ingress controller, I wanted to verify that this use is correct. It looks like the variable names are the same too.